.github/workflows/test.yml

```diff
@@ -35,8 +35,14 @@ jobs:
         with:
           nix_path: nixpkgs=channel:nixos-20.09
       - uses: cachix/install-nix-action@v12
-        name: Nix unstable base
+        name: Nix 21.05 base
         if: matrix.ghc == '9.0.1'
+        with:
+          nix_path: nixpkgs=channel:nixos-21.05
+      - uses: cachix/install-nix-action@v12
+        name: Nix unstable base
+        # if: matrix.ghc == '9.0.1'
+        if: false
```

Comment:
Can this `if: false` step be removed?

Reply:
I was using that to essentially disable that step, but leave it around since I suspect it will be useful in the future.

Reply:
I could also comment out that block if you think that's better.

Reply:
Ah, I wasn't aware that that was intentional. In that case, I'm fine with leaving it in, but it would be worth leaving a comment saying why it's there.